docs/reference/cat/dataframeanalytics.asciidoc

```diff
@@ -0,0 +1,165 @@
+[role="xpack"]
+[testenv="platinum"]
+[[cat-dfanalytics]]
+=== cat {dfanalytics} API
+++++
+<titleabbrev>cat {dfanalytics}</titleabbrev>
+++++
+
+Returns configuration and usage information about {dfanalytics-jobs}.
+
+
+[[cat-dfanalytics-request]]
+==== {api-request-title}
+
+`GET /_cat/ml/data_frame/analytics/<data_frame_analytics_id>` +
+
+`GET /_cat/ml/data_frame/analytics`
+
+
+[[cat-dfanalytics-prereqs]]
+==== {api-prereq-title}
+
+If the {es} {security-features} are enabled, you must have the following 
+privileges:
+
+* cluster: `monitor_ml`
+  
+For more information, see <<security-privileges>> and <<built-in-roles>>.
```

Comment:
You don't mention any built-in roles, so I think it's more useful to link to the full list of privileges required by ML:
```suggestion
For more information, see <<security-privileges>> and {ml-docs}/setup.html[Set up {ml-features}].
```